test/smoke/testApps/OpenTelemetryApiSupport/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/OpenTelemetryApiSupportTest.java

```diff
@@ -0,0 +1,38 @@
+package com.microsoft.applicationinsights.smoketest;
+
+import java.util.List;
+
+import com.microsoft.applicationinsights.internal.schemav2.Data;
+import com.microsoft.applicationinsights.internal.schemav2.Envelope;
+import com.microsoft.applicationinsights.internal.schemav2.RemoteDependencyData;
+import com.microsoft.applicationinsights.internal.schemav2.RequestData;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```

Comment:
can we get rid of the wildcard? all you need are assertEquals and assertTrue.

Reply:
this is convention in all the smoke tests, if we want to change we should do across all